FC-0068 docs: Documentation Templates (How-to, Reference, Concept) (#619)
openedx/docs.openedx.org: source/documentors/references/templates/doc_concept_template.rst

Reviewer: @Apgomeznext there is a punctuation error (last full stop) which should be fixed, and also there is an extra space between brackets:
Lines 6 should be like this: For example, “What is an XBlock? The Open edX platform provides different components, called XBlocks, that can work to create a course, like text, video, assessment, and discussions.” (Replace this text with your content).

Concept Document Template
#########################

A concept is a document that clarifies and illuminates a particular topic. It is understanding-oriented. It could be considered a conceptual guide. This type of document answers a why question. It could be descriptive, historical, or even propose different alternatives to explain the bigger picture and give context. 

> For example, “What is an XBlock? The Open edX platform provides different components, called XBlocks, that can work to create a course, like text, video, assessment, and discussions.”  (Replace this text with your content.)

Reviewer 2: ```suggestion
For example, “What is an XBlock? The Open edX platform provides different components, called XBlocks, that can work to create a course, like text, video, assessment, and discussions.”  (Replace this text with your content).
```

> Since this type of document seeks to explain a topic, you will likely need to add images to exemplify the topic you are talking about. For example:

Reviewer 2: ```suggestion
Since this type of document seeks to explain a topic, you will likely need to add images to illustrate the topic you are talking about. For example:
```

"exemplify" is a more complicated english word than "illustrate"

> .. image:: /_images/documentors_howto/image_example.png

Reviewer 2: Why is the example text above about XBlocks, but this image unrelated to XBlocks? It's confusing. I'd make the image and the example text above at line 6 match.

	:width: 500px
	:align: center
	:alt: This image example shows how to access the GitHub repository from the Open edX documentation repository.

External Links
--------------

> You can have `inline links <https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#linking>`_ and `indirect links`_, that are useful if you want to link to the same thing multiple times, or if the URL is really long and you want things to read more cleanly.

Reviewer 2: ```suggestion
You can use `inline links <https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#linking>`_ or `indirect links`_. Indirect links are useful if you want to link to the same thing multiple times, or if the URL is really long and you want the raw RST to read more cleanly.
```

folder.

By default, it will use the title of the doc as the link text but you can override that with :doc:`other text <doc_how_to_template>` if you want.

RST Template
************

.. code-block:: RST

	Concept Document Template
	#########################

	A concept is a document that clarifies and illuminates a particular topic. It is understanding-oriented. It could be considered a conceptual guide. This type of document answers a why question. It could be descriptive, historical, or even propose different alternatives to explain the bigger picture and give context. 

	For example, “What is an XBlock? The Open edX platform provides different components, called XBlocks, that can work to create a course, like text, video, assessment, and discussions.”  (Replace this text with your content.)

	Since this type of document seeks to explain a topic, you will likely need to add images to exemplify the topic you are talking about. For example:

	.. image:: /_images/documentors_howto/image_example.png
		:width: 500px
		:align: center
		:alt: This image example shows how to access the GitHub repository from the Open edX documentation repository.

	External Links
	--------------

> 	You can have `inline links <https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#linking>`_ and `indirect links`_, that are useful if you want to link to the same thing multiple times, or if the URL is really long and you want things to read more cleanly.

Reviewer 2: ```suggestion
	You can use `inline links <https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#linking>`_ or `indirect links`_. Indirect links are useful if you want to link to the same thing multiple times, or if the URL is really long and you want the raw RST to read more cleanly.
```

A concept is a document that clarifies and illuminates a particular topic. It is understanding-oriented. It could be considered a conceptual guide. This type of document answers a why question. It could be descriptive, historical, or even propose different alternatives to explain the bigger picture and give context. 

> For example, “What is an XBlock? The Open edX platform provides different components, called XBlocks, that can work to create a course, like text, video, assessment, and discussions”.

Reviewer 2: My comment about the image example was that this line ("What is an XBlock") might be clearer if the associated image _also_ was about the same thing. So you changed the image to one about Superset, which is still unrelated to this line about XBlocks. It's not a big deal but that was what I was saying.

Author: You are right. Sorry about the confusion.
